components/cards/basic-cards/basic-cards.css

```diff
@@ -0,0 +1,206 @@
+}
+
+/* styling all the general siblings of .ed_card-subcontainer1 */
+.et_b-card-subcontainer1~* {
+    font-family: 'Monoton', cursive;
+    font-family: 'Nunito', sans-serif;
+    font-family: 'Rubik', sans-serif;
+    font-family: 'Sen', sans-serif;
+    margin-left: 0.6rem;
+    margin-right: 0.6rem;
+}
+
+.et_b-card-subcontainer1 {
+    background-image: url(https://wallpaperaccess.com/full/3846486.jpg);
+    background-size: cover;
+    border-top-left-radius: inherit;
+    border-top-right-radius: inherit;
+    width: 100%;
+    height: 50%;
+}
+
+.et_b-card-subcontainer2 {
+    background-color: rgb(235, 245, 245);
+}
+
+.et_b-subcontainer2-heading1 {
+    font-size: 1.1rem;
+}
+
+.et_b-subcontainer2-heading2 {
+    font-size: 0.8rem;
+    color: rgb(101, 101, 107);
+}
+
+.et_b-subcontainer3-para {
+    font-size: 0.7rem;
+    color: rgb(94, 85, 85);
+}
+
+.et_b-card-subcontainer4 {
+    display: flex;
+    margin-top: 1.3rem;
+}
+
+.et_b-subcontainer4-action-buttons button {
+    border: none;
+    color: rgb(69, 100, 204);
+    font-size: 0.7rem;
+    font-weight: 600;
+    padding-right: 0.8rem;
+}
+
+.et_b-subcontainer4-icon-buttons {
+    display: flex;
+    flex-grow: 1;
+    justify-content: space-evenly;
+}
+
+.et_b-subcontainer4-icon-buttons button {
+    border: none;
+    color: rgb(94, 85, 85);
+}
+
+button span{
+    display: none;
+    position: relative;
+}
+
+button:hover span{
+    background-color: rgb(12, 12, 12);
+    border-radius: 1.2rem;
+    color: rgb(250, 244, 244);
+    display: inline;
+    font-family: inherit;
+    padding: 0.8rem;
+    position: absolute;
+    opacity: 0.7;
+}
+
+button:hover {
+    color: rgb(212, 33, 33);
+    cursor: pointer;
+}
+
+[class^="et"]:hover{
+    cursor: pointer;
+}
+/* basic component styling ends */
+
+/* basic w/ text over media component2 starts */
+.et_basic-wtom-card-container {
+    height: 240px;
+    position: relative;
+}
+
+.et_basic-wtom-card-subcontainer1 {
+    height: 60%;
+}
+
+.et_basic-wtom-card-subcontainer2 {
+    background-color: transparent;
+    position: absolute;
+    top: 40%;
+}
+
+.et_basic-wtom-card-subcontainer2>* {
+    color: white;
+}
+
+/* basic w/ text over media component2 ends */
+
+/* basic w/ header componenet3 starts */
+.et_b-wheader-card-container {
+    display: flex;
+    flex-direction: column;
+}
+
+.et_b-wheader-card-subcontainer2 {
+    order: -1;
+    margin-top: 0.7rem
+}
+
+.et_b-wheader-card-subcontainer1 {
+    border-radius: 0;
+}
+
+/* basic w/ header component3 ends */
+
+/* basic w/ only action buttons component4 starts */
+.et_basic-woab-subcontainer4-icon-buttons {
+    display: none;
+}
+
+/* basic w/ only action buttons component4 ends */
+
+/* basic w/ only icons component5 starts */
+.et_basic-woi-subcontainer4-action-buttons {
+    visibility: hidden;
+}
+
+/* basic w/ only icons component5 ends */
+
+/* basic w/ horizontal image and text component6 starts */
+.et_basic-whiat-card-container{
+    height: 120px;
```

Comment:
use same units for consistency

Reply:
okay will look into it!! thankyou so much for reviewing!!